src/server/server-html.jsx

```diff
@@ -212,6 +218,19 @@ export default function ServerHTML({
                 {assets.script.map((href, idx) => (
                     <script key={idx} src={href} />
                 ))}
+                {gptEnabled ? (
+                    <script
```

Comment:
async=false?

Reply:
The browser loads scripts synchronously unless you specify `async=true`. BSA was telling me not to add that tag.

Info here if you want to read: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script